backend/src/main/java/gov/cdc/usds/simplereport/api/export/DataHubUploadController.java

```diff
@@ -0,0 +1,49 @@
+package gov.cdc.usds.simplereport.api.export;
+
+import gov.cdc.usds.simplereport.service.DataHubUploaderService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+
+/**
+ * Created by tomn on 11Dec2020
+ */
+@RestController
+@RequestMapping(value = "/export")
+@Validated
+public class DataHubUploadController {
+
+    private final DataHubUploaderService _hubuploadservice;
+
+    public DataHubUploadController(DataHubUploaderService us) {
+        this._hubuploadservice = us;
+    }
+
+    @GetMapping(value = "/uploadTestEvent")
+    public String uploadTestEventCSVToDataHub(@RequestParam String apikey,
+                                              @RequestParam(defaultValue = "") String startupdateby
```

Comment:
Spring will parse a date for you:

```
	@RequestParam(required=false) @DateTimeFormat(iso=ISO.DATE_TIME) ZonedDateTime startUpdateBy,
```

Reply:
Thanks. I'll switch to that.

Reply:
Opened #392 to make a 2nd pass and Spring-ify the code.